extension/test/event-page-data-webtest.js

```diff
@@ -27,7 +27,8 @@ describe('EventPage', function() {
       ]);
       assert.isOk(result);
       assert.isOk(
-          result.hasOwnProperty('urla'),
+        // eslint-disable-next-line no-prototype-builtins
+        result.hasOwnProperty('urla'),
```

Comment:
Paul, have you considered pros and cons and decided that you want the environment where Object prototype functions are disallowed? Or are we just following what the tool tells us?

The alternative is verbose: `Object.prototype.hasOwnProperty.call(obj, 'prop')`.

I have never heard anyone complaining or being bitten by a lack of prototype, so this appears to me like causing more discomfort than helping.

What is your take on this?

Reply:
Sorry for piling on, but yeah, what Piotr said.

Reply:
I'm going to disable this  since it appears to be low value.  It can always be enabled later.